db/migrations/mysql/000021_create_audits.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS audits (
+    Id varchar(26) NOT NULL,
+    CreateAt bigint(20) DEFAULT NULL,
+    UserId varchar(26) DEFAULT NULL,
+    Action text,
+    ExtraInfo text,
+    IpAddress varchar(64) DEFAULT NULL,
+    SessionId varchar(26) DEFAULT NULL,
+    PRIMARY KEY (Id),
+    KEY idx_audits_user_id (UserId)
```

Comment:
We should move this out of the create table. Will do it as part of the review and merge to the schemas branch.